src/game.cpp

```diff
@@ -9285,7 +9285,7 @@ int game::list_items(const int iLastState)
     bool addcategory = false;
     int iPage = 0;
     int iCatSortNum = 0;
-    item activeItem;
+    map_item_stack *activeItem;
```

Comment:
I'm thinking this broke the build: Jenkins/clang is throwing a maybe_uninitialized error.  Will see about a bandaid but might want to look into a better fix.